doc/release-notes/release-8.sgml.in

```diff
@@ -39,6 +39,8 @@ The Squid-@SQUID_RELEASE@ change history can be <url url="https://github.com/squ
 	<p>Removed <em>ext_lm_group_acl</em> external acl helper. No product
 	   currently on the market suports the LanManager protocol
 	   in their default configuration.
+	<p>Removed <em>ntlm_smb_lm_auth</em> NTLM authentication helper.
+	   Use the <em>html_auth</em> helper from the Samba project instead.
```

Comment:
```suggestion
	   Use the <em>ntlm_auth</em> helper from the Samba project instead.
```

I may be worth highlighting that this removed helper only supported protocols from Windows older than Win95 when they are wrapped in NTLMSSP for HTTP Authentication. It does not support actual NTLM.

Reply:
> It does not support actual NTLM.

I adjusted PR description to remove "implementation of NTLM" and to add one more reason for relying on Samba tooling. Please undo/revert if you do not like those adjustments; I do not insist on them.